test/e2e/predictor/test_mlflow.py

```diff
@@ -0,0 +1,66 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from kubernetes import client
+from kserve import (
+    constants,
+    KServeClient,
+    V1beta1InferenceService,
+    V1beta1InferenceServiceSpec,
+    V1beta1PredictorSpec,
+    V1beta1ModelSpec,
+    V1beta1ModelFormat,
+)
+from kubernetes.client import V1ResourceRequirements
+
+from ..common.utils import predict
+from ..common.utils import KSERVE_TEST_NAMESPACE
+
+
+def test_mlflow_v2_runtime_kserve():
+    service_name = "isvc-mlflow-v2-runtime"
+
+    predictor = V1beta1PredictorSpec(
+        min_replicas=1,
+        model=V1beta1ModelSpec(
+            model_format=V1beta1ModelFormat(
+                name="mlflow",
+            ),
+            runtime="kserve-mlserver",
```

Comment:
I think we shouldn't need to specify the runtime here as MLServer is the only runtime that can serve mlflow model